objcopy: add page (#12754)
tldr-pages/tldr: pages/linux/objcopy.md

# objcopy

> The GNU objcopy utility copies the contents of an object file to another.
> More information: <https://manned.org/objcopy>.

> - Copy data to another file:

Reviewer: ```suggestion
- Copy all data from a specific file to another one:
```

> `objcopy {{/your/path/to/file/src}} {{/your/path/to/file/destination}}`

Reviewer: ```suggestion
`objcopy {{path/to/source_file}} {{path/to/target_file}}`
```
The [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides%2Fstyle-guide.md) recommends:
- Starting placeholders for **absolute** path arguments with "/" and _**not** starting with "/" for **relative** path arguments_
- File placeholders in the form "{{path/to/file}}", where file can be relaced with a more descriptive name

- Translate object files from one format to another:

> `objcopy --input-target={{inputformat}} --output-target={{output_format}} {{/path/to/your/src}} {{path/to/your/destination}}`

Reviewer: ```suggestion
`objcopy --input-target={{inputformat}} --output-target={{output_format}} {{path/to/source_file}} {{path/to/destination_file}}`
```
See https://github.com/tldr-pages/tldr/pull/12754#discussion_r1592958781.

> - Strip all symbol information from file:

Reviewer: ```suggestion
- Strip all symbol information from a specific file:
```

> `objcopy --strip-all {{/path/to/your/src}} {{path/to/your/destination}}`

Reviewer: ```suggestion
`objcopy --strip-all {{path/to/source_file}} {{path/to/destination_file}}`
```
See https://github.com/tldr-pages/tldr/pull/12754#discussion_r1592958781.

- Strip debugging information from file:

> `objcopy --strip-debug {{/path/to/your/src}} {{path/to/your/destination}}`

Reviewer: ```suggestion
`objcopy --strip-debug {{path/to/source_file}} {{path/to/destination_file}}`
```
See https://github.com/tldr-pages/tldr/pull/12754#discussion_r1592958781.

> - Copy a specific section from the source to destination:

Reviewer: ```suggestion
- Copy a specific section of the source file to the destination file:
```

> `objcopy --only-section={{section}} {{/path/to/your/src}} {{path/to/your/destination}}`

Reviewer: ```suggestion
`objcopy --only-section={{section}} {{/path/to/source_file}} {{path/to/destination_file}}`
```
See https://github.com/tldr-pages/tldr/pull/12754#discussion_r1592958781.
